sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallingServerClientBuilder.java

```diff
@@ -341,4 +350,8 @@ private HttpLogOptions getHttpLogOptions() {
 
         return httpLogOptions;
     }
+
+    private String getHostNameFromEndpoint() {
+        return endpoint.replace("https://", "");
```

Comment:
are we 100% sure that the endpoint will always have `https`?

Reply:
Good point, we probably also want to check for http. Anything else to look for?

Reply:
is there a lib in java like URL in C# that does this for us?

Reply:
Yeah, I recommend you to do as Bharat suggests. Let the library do the heavy lifting.
There is URL library where you can create the URL from a String and call `getHost()` to get the host name.

Reply:
Thanks, updated.